docs/deployment/FPC_on_Azure.md

```diff
@@ -0,0 +1,139 @@
+# How to use an Azure Confidential Computing Instance to run FPC in HW mode
+Date of writing 14.03.2022.
+This guide is based on the following sources.
+[Koshi Ikegawa](https://qiita-com.translate.goog/ikegawa-koshi/items/8cf1fef1004fc16beb15?_x_tr_sl=ja&_x_tr_tl=en&_x_tr_hl=de&_x_tr_pto=wapp#fpc%E3%81%AE%E3%83%93%E3%83%AB%E3%83%89)
+
+- [How to use an Azure Confidential Computing Instance to run FPC in HW mode](#how-to-use-an-azure-confidential-computing-instance-to-run-fpc-in-hw-mode)
```

Comment:
I personally would remove the first line :)

Reply:
Which line are you referring to?
The Date of writing or the reference to the article?

Reply:
sorry - first bullet point.

Reply:
I stand on the line ;)

So you mean the table of contents. I am confused because there is no bullet point besides the TOC

Reply:
I was suggesting to remove the first Entry of the TOC. (- [How to use an Azure Confidential Computing Instance to run FPC in HW mode](#how-to-use-an-azure-confidential-computing-instance-to-run-fpc-in-hw-mode))

Reply:
Ah sorry.
Now, I get it ;)